docs/01-app/03-api-reference/05-config/01-next-config-js/inlineCss.mdx

```diff
@@ -4,6 +4,8 @@ description: Enable inline CSS support.
 version: experimental
 ---
 
+# Usage
```

Comment:
might be too large to use h1, I think it's common to have h2 for those sections.

Reply:
Agreed!